off-chain/test/Examples/Addition/SeathSetup.purs

```diff
@@ -102,19 +104,40 @@ submitChain leader participants txs log = do
     log
     -- logInfo' $ "submiting: " <> show balancedAndSignedTransaction
     transactionId <- submit balancedAndSignedTransaction
-    awaitTxConfirmed transactionId
+    -- awaitTxConfirmed transactionId
     -- logInfo' $ "submited: " <> show transactionId
     pure $ transactionId
 
+newtype BlockhainState = BlockhainState
+  { leaderUTXOs :: Maybe UtxoMap
```

Comment:
Hmm, didn't think about that. I just used the same type that `getWalletUtxos` from CTL returns, but not sure atm what is the case for `Notning` there.

Reply:
for `getWalletUxos` make sense since is different not having any `UTxO` than failing to query it. In this case we control if there are `UTxOs` in the leader, if we can't get them, we did something wrong.